File: 2st/Riyeon/합성수 찾기.py
def solution(n):
    if n < 4:
        return 0
    
    answer=0
    for i in range(4, n+1):
        cnt=0
        for j in range(1, i+1):
            if i%j == 0:
                cnt += 1
        if cnt >= 3:
            answer+=1
    return answer

File: 4st/Eunseong/합성수 찾기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        int cnt = 0;
        for (int i = 1; i <= n; i++) {
            cnt = 0;
            for (int j = 1; j <= i; j++) {
                if (i % j == 0) cnt++;
                if (cnt >= 3) {
                    answer++;
                    break;
                }
            }
        }
        return answer;
    }
}